File: src/main/java/com/oyosite/ticon/specutils/SpectralUtilitiesClient.java
package com.oyosite.ticon.specutils;

import com.oyosite.ticon.specutils.block.*;
import net.fabricmc.api.*;

@Environment(EnvType.CLIENT)
public class SpectralUtilitiesClient implements ClientModInitializer {

	@Override
	public void onInitializeClient() {
		BlockRegistry.registerClient();
	}
}

File: src/main/kotlin/com/oyosite/ticon/specutils/SpectralUtilitiesClient.kt
package com.oyosite.ticon.specutils

import com.oyosite.ticon.specutils.block.BlockRegistry
import net.fabricmc.api.ClientModInitializer
import net.fabricmc.api.EnvType
import net.fabricmc.api.Environment

@Environment(EnvType.CLIENT)
class SpectralUtilitiesClient : ClientModInitializer {
    override fun onInitializeClient() {
        BlockRegistry.registerClient()
    }
}